### PR TITLE
chore: update the merge-me-action configuration to support dependabot [skip release]

### PR DESCRIPTION
**Pull request by acazacu**

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 version: 2
 updates:
-  - directory: '/'
+  - commit-message:
+      prefix: chore
+      include: scope
+    directory: '/'
     package-ecosystem: github-actions
```
Comment (merlinnot) [inline .github/dependabot.yaml:7]: Also `open-pull-requests-limit: 99`?

Comment (acazacu) [reply .github/dependabot.yaml:7]: I found this kind of outlandish. If we ever get there, we have a big big problem. But alright, I'll add it!

**Comment by acazacu** [comment]
> Wait, why the scope is `skip-release`?

Because it does not change to code or functionality. It is a CI change only. Which part of this update would warrant a release?

**Comment by merlinnot** [comment]
> > Wait, why the scope is `skip-release`?
> 
> Because it does not change to code or functionality. It is a CI change only. Which part of this update would warrant a release?

Adding it in the scope is clearly against conventional commits. From https://www.conventionalcommits.org/en/v1.0.0/:

> A scope may be provided to a commit’s type, to provide additional contextual information

This doesn't add any value in release notes and the changelog.

 Why would it hurt to release it?

**Comment by mad-it** [comment]
> This doesn't add any value in release notes and the changelog.
> 
> Why would it hurt to release it?

Because it adds no value to the package that its trying to release. "We updated our CI" should not be in the changelog as it does not influence any functionality of the package that we are providing

**Comment by merlinnot** [comment]
So it doesn't hurt? If it does nothing, then it can't hurt, right?

**Comment by acazacu** [comment]
> So it doesn't hurt? If it does nothing, then it can't hurt, right?

I like where this is going 😬 . If it does nothing, why waste energy doing it?

In a completely different argument, where this does actually happen, I understand that we should not use scope for this as it is non-conformant with conventional  commits. How should we deal with this then?

I see great utility in having the ability to skip a release. Especially in libraries whose releases trigger downstream deployments. Deployments can be a heavy and risky operation, performing them should always come with good reason.

**Comment by merlinnot** [comment]
Most importantly, one should never feel any risk when performing a deployment. If that's the case, the issue is elsewhere. Infrastructure and applications should be perfectly reliable and that's why at @ridedott all applications are deployed continuously, multiple times a day.

Getting back to your question, I'm not against skipping releases if a developer wants to do that. Free choice. Scope is however not the right place for it, as it is an important part of conventional commits.

By convention, it is often seen that `[skip ci]` is added as a suffix to the message. Many CI providers interpret that as an ask to don't run CI jobs. We need to always run CI in the master branch for code coverage comparison between branches, so that's not an option. We could however have a similar convention with `[skip release]` (as a suffix, with brackets).

**Comment by acazacu** [comment]
> Most importantly, one should never feel any risk when performing a deployment. If that's the case, the issue is elsewhere. Infrastructure and applications should be perfectly reliable and that's why at @ridedott all applications are deployed continuously, multiple times a day.
> 
> Getting back to your question, I'm not against skipping releases if a developer wants to do that. Free choice. Scope is however not the right place for it, as it is an important part of conventional commits.
> 
> By convention, it is often seen that `[skip ci]` is added as a suffix to the message. Many CI providers interpret that as an ask to don't run CI jobs. We need to always run CI in the master branch for code coverage comparison between branches, so that's not an option. We could however have a similar convention with `[skip release]` (as a suffix, with brackets).

Ok, [skip release] sounds good to me.

File: .github/workflows/continuous-delivery.yaml [hunk 2]
```diff
@@ -49,7 +49,7 @@ jobs:
               { "release": "patch", "type": "ci" },
               { "release": "patch", "type": "improvement" },
               { "release": "patch", "type": "refactor" },
-              { "release": false, "scope": "skip-release" }
```
Comment (merlinnot) [inline .github/workflows/continuous-delivery.yaml:52]: Ugh, didn't even notice that before 😢

File: .github/workflows/continuous-integration.yaml [hunk 3]
```diff
@@ -129,7 +129,7 @@ jobs:
       - name: Types
         run: npm run types
   merge-me:
-    name: Merge me!
+    name: Auto merge Dependabot updates
```
Comment (merlinnot) [inline .github/workflows/continuous-integration.yaml:132]: That's incorrect.

File: .github/workflows/continuous-integration.yaml [hunk 4]
```diff
@@ -138,7 +138,7 @@ jobs:
       - types
     runs-on: ubuntu-latest
     steps:
-      - name: Merge me!
-        uses: ridedott/merge-me-action@master
+      - name: Auto merge
+        uses: ridedott/dependabot-auto-merge-action@master
```
Comment (merlinnot) [inline .github/workflows/continuous-integration.yaml:142]: These changes are incorrect.

File: .github/workflows/continuous-integration.yaml [hunk 5]
```diff
@@ -138,7 +138,7 @@ jobs:
...
         with:
```
Comment (merlinnot) [inline .github/workflows/continuous-integration.yaml:143]: We should specify a new bot name.

Comment (acazacu) [reply .github/workflows/continuous-integration.yaml:143]: This is taken from the firestore-schema repository. Which I assumed would have a correct implementation of this. If this changed. I'm looking at https://github.com/ridedott/merge-me-action.

I see nothing wrong about how this was setup before I made these changes.

Comment (merlinnot) [reply .github/workflows/continuous-integration.yaml:143]: We made a switch from dependabot-preview to dependabot, so we must use https://github.com/ridedott/merge-me-action#enable-auto-merge-for-a-different-bot. See https://github.com/ridedott/eslint-config/blob/master/.github/workflows/continuous-integration.yaml#L100.

Comment (acazacu) [reply .github/workflows/continuous-integration.yaml:143]: I reverted the changes and updated the configuration. Reading [this](https://github.com/ridedott/merge-me-action#enable-auto-merge-for-a-different-bot) documentation and looking at Dependabot's PR's, it is not really clear that the username is wrong.

At first glance, it looks as if Dependabot's PR's are authored by `dependabot`, without `bot`. `bot` does not seem to be part of the username.

**Pull request merged**